patches/api/0422-Use-ApiStatus.Internal-instead-of-Deprecated.patch

```diff
@@ -0,0 +1,700 @@
+      *
+      * @param name the entity type's name
+      * @return the matching entity type or null
+-     * @deprecated Magic value
++     * @apiNote Internal Use Only
+      */
+-    @Deprecated
++    @org.jetbrains.annotations.ApiStatus.Internal // Paper
+     @Contract("null -> null")
+     @Nullable
+     public static EntityType fromName(@Nullable String name) {
+@@ -406,9 +406,9 @@ public enum EntityType implements Keyed, Translatable, net.kyori.adventure.trans
+      *
+      * @param id the raw type id
+      * @return the matching entity type or null
+-     * @deprecated Magic value
++     * @apiNote Internal Use Only
+      */
+-    @Deprecated
++    @org.jetbrains.annotations.ApiStatus.Internal // Paper
+     @Nullable
+     public static EntityType fromId(int id) {
+         if (id > Short.MAX_VALUE) {
+diff --git a/src/main/java/org/bukkit/entity/Ocelot.java b/src/main/java/org/bukkit/entity/Ocelot.java
+index 35f970ca47de48992d4beb197695126f71d005da..61f7ccc5673d826dd25589662f58485fe21dd069 100644
+--- a/src/main/java/org/bukkit/entity/Ocelot.java
++++ b/src/main/java/org/bukkit/entity/Ocelot.java
+@@ -71,9 +71,9 @@ public interface Ocelot extends Animals {
+          * Gets the ID of this cat type.
+          *
+          * @return Type ID.
+-         * @deprecated Magic value
++         * @apiNote Internal Use Only
+          */
+-        @Deprecated
++        @org.jetbrains.annotations.ApiStatus.Internal // Paper
+         public int getId() {
+             return id;
+         }
+@@ -83,9 +83,9 @@ public interface Ocelot extends Animals {
+          *
+          * @param id ID of the cat type to get.
+          * @return Resulting type, or null if not found.
+-         * @deprecated Magic value
++         * @apiNote Internal Use Only
+          */
+-        @Deprecated
++        @org.jetbrains.annotations.ApiStatus.Internal // Paper
+         @Nullable
+         public static Type getType(int id) {
+             return (id >= types.length) ? null : types[id];
+diff --git a/src/main/java/org/bukkit/entity/TextDisplay.java b/src/main/java/org/bukkit/entity/TextDisplay.java
+index f101ffde9e7b3f6c3ab6f1cffb8d6f4e720634b2..321024b5f9aeb6ecbcd85db3717f783a6debc95f 100644
+--- a/src/main/java/org/bukkit/entity/TextDisplay.java
++++ b/src/main/java/org/bukkit/entity/TextDisplay.java
+@@ -62,19 +62,17 @@ public interface TextDisplay extends Display {
+      * Gets the text background color.
+      *
+      * @return the background color
+-     * @deprecated API subject to change
+      */
+     @Nullable
+-    @Deprecated
++    // @Deprecated // Paper - is stable API
+     Color getBackgroundColor();
+ 
+     /**
+      * Sets the text background color.
+      *
+      * @param color new background color
+-     * @deprecated API subject to change
+      */
+-    @Deprecated
++    @Deprecated // Paper - is stable API
+     void setBackgroundColor(@Nullable Color color);
```

Comment:
This annotation doesn't seem to be commented out like the one above is.

Reply:
good catch, corrected.